hardware/Limitless.cpp

```diff
@@ -238,15 +238,17 @@ bool CLimitLess::StartHardware()
 				{
 					if (!AddSwitchIfNotExits(3, "AppLamp Group3"))
 					{
-						AddSwitchIfNotExits(4, "AppLamp Group4");
+						if (!AddSwitchIfNotExits(4, "AppLamp Group4"))
+						{
+							if (m_BridgeType == LBTYPE_V6)
```

Comment:
Will do, this was not a change on purpose but from a behind fork. Will send new pull request later.